syntax = "proto3";

package ommx.v1;

// A message representing a one-hot constraint.
message OneHot {
  // The list of ids of decision variables that are constrained to be one-hot.
  repeated uint64 decision_variables = 1;
}
